testsuite/WIKI.md

```diff
@@ -0,0 +1,198 @@
+The unit tests and the test data are bundled together in the package **MDAnalyisTests-_release_**. In order to run the tests, this package must be installed in addition to MDAnalysis.
+
+The tests also rely on the `pytest` and `numpy` packages, and require both to run.
+
+# Users
+Install [MDAnalysisTests](MDAnalysisTests) via
+```
+pip install --upgrade MDAnalysisTests
+```
+or download the tar file, unpack, and run `python setup.py install`.
+
+Run the tests by invoking
+```
+python -c 'from MDAnalysis.tests import test; test()'
+```
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error [ask on the user mailing list](http://users.mdanalysis.org) or [raise an issue](/MDAnalysis/mdanalysis/issues).
+
+# Developers #
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error **fix your code** (or *raise an issue*).
+
+**Do not push code that fails to the development branch.** Instead, push it to a feature or issue branch. It will run automatically through the unit tests by travis-ci and it will be available for comment and discussion by other developers.
+
+## Recommended ##
+Use the tests from the [git source repository](Source), which are located in the [testsuite/MDAnalysisTests](https://github.com/MDAnalysis/mdanalysis/tree/develop/testsuite) directory:
+```
+cd testsuite/MDAnalysisTests
+pytest  --disable-pytest-warnings --numprocesses 4
```

Comment:
I would leave the `--disable-pytest.warnings` flag. The added noise at the end of the tests is immense, and floods whatever might of use (right now I get over 400 warnings only for the "Failed to guess the mass for...").

I would also vote for the use of `--pyargs MDAnalysisTests`. This allows the tests to be run from anywhere, instead of having to cd into the tests directory and running from there.